roles/confluent.ksql/tasks/rbac.yml

```
@@ -144,3 +148,35 @@
       }
     status_code: 204
   when: "'schema_registry' in groups"
+
+- name: Grant ksql user the ResourceOwner role on the Processing Log Topic
+  uri:
+    url: "{{mds_http_protocol}}://{{ groups['kafka_broker'][0] }}:{{mds_port}}/security/1.0/principals/User:{{item}}/roles/ResourceOwner/bindings"
+    method: POST
+    validate_certs: false
+    force_basic_auth: true
+    url_username: "{{mds_super_user}}"
+    url_password: "{{mds_super_user_password}}"
+    headers:
+      Content-Type: application/json
+    body_format: json
+    body: >
+      {
+        "scope": {
+          "clusters": {
+            "kafka-cluster":"{{kafka_cluster_id}}"
+          }
+        },
+        "resourcePatterns": [
+          {
+            "resourceType":"Topic",
+            "name":"{{ksql.properties['ksql.service.id']}}processing_log",
+            "patternType":"LITERAL"
+          }
+        ]
+      }
+    status_code: 204
+    loop:
+      - "{{ksql_ldap_user}}"
+      - "{{ksql_log4j_principal}}"
+    when: ksql_log_streaming_enabled | bool
```

```suggestion
  loop:
    - "{{ksql_ldap_user}}"
    - "{{ksql_log4j_principal}}"
  when: ksql_log_streaming_enabled | bool
```
looks like the indentation needs to go one back